### PR TITLE
docs: update contributing guide

### PR DESCRIPTION
**Pull request by SukkaW**
## Check List

**Please read and check followings before submitting a PR.**

- [ ] I want to publish my theme on Hexo official website.
    - [ ] I have read the [theme publishing doc](https://hexo.io/docs/themes#Publishing).
    - [ ] `name` is unique.
    - [ ] `link` URL is correct.
    - [ ] `preview` URL is correct.
    - [ ] `preview` URL web site is rendered correctly.
    - [ ] Add a screenshot to `source/themes/screenshots`.
    - [ ] Screenshot filename is same as value of `name`.
    - [ ] Screenshot size is `800 * 500`.
    - [ ] Screenshot file format is `png`.
- [ ] I want to publish my plugin on Hexo official website.
    - [ ] I have read the [plugin publishing doc](https://hexo.io/docs/plugins#Publishing).
    - [ ] `name` is unique.
    - [ ] `link` URL is correct.
- [x] Others (Update, fix, translation, etc...)

- Add code of conduct
- Add eslint-config-hexo

### CODE REVIEW

File: source/zh-tw/docs/contributing.md [hunk 1]
```diff
@@ -92,8 +97,7 @@ Hexo 文件開放原始碼，您可以在 [hexojs/site] 找到原始碼。
 當您使用 Hexo 遭遇問題時，可試著在 [解決問題](troubleshooting.html) 中尋找解答，或是在 [GitHub](https://github.com/hexojs/hexo/issues) 或 [Google Group](https://groups.google.com/group/hexo) 詢問。若找不到答案，請至 GitHub 回報。
 
 1. 以 [除錯模式](commands.html#除錯模式) 再執行一次。
-2. 執行 `hexo vesion` 並檢查版本資訊。
-3. 把除錯資訊和版本資訊都貼到 GitHub。
+2. 在 GitHub 上提交新 Issue 時，請按照問題模板中的步驟提供調試消息和版本資訊。
```
Comment (tommy351) [inline source/zh-tw/docs/contributing.md]:
```suggestion
2. 在 GitHub 上提交新 Issue 時，請按照問題模板中的步驟提供除錯資訊和版本資訊。
```

File: source/zh-tw/docs/contributing.md [hunk 2]
```diff
@@ -10,36 +10,40 @@ We welcome you to join the development of Hexo. 🤗
...
 
+Please read [Contributor Covenant Code of Conduct](https://github.com/hexojs/hexo/blob/master/CODE_OF_CONDUCT.md) first.
+
 請遵守以下準則：
 
 - 遵守 [Google JavaScript 代碼風格](https://google.github.io/styleguide/jsguide.html)。
 - 使用 2 個空格縮排。
 - 不要把逗號放在最前面。
 
+Also, Hexo has its own [ESLint config](https://github.com/hexojs/eslint-config-hexo), so please make sure your contribution will make ESLint happy.
```
Comment (tommy351) [inline source/zh-tw/docs/contributing.md]: 此外，Hexo 有 [ESLint 設定](https://github.com/hexojs/eslint-config-hexo)，因此請務必確認您的貢獻能夠通過 ESLint。

Comment (SukkaW) [inline source/zh-tw/docs/contributing.md]:
```suggestion
此外，Hexo 有 [ESLint 設定](https://github.com/hexojs/eslint-config-hexo)，因此請務必確認您的貢獻能夠通過 ESLint。
```

File: source/zh-cn/docs/contributing.md [hunk 3]
```diff
@@ -10,36 +10,40 @@ We welcome you to join the development of Hexo. 🤗
...
+
+请使用以下代码风格：
 
 - 遵守 [Google JavaScript 代码风格](https://google.github.io/styleguide/jsguide.html)。
-- 使用 2 个空格缩排。
+- 缩进使用 2 个空格。
 - 不要把逗号放在最前面。
 
-### 工作流程
+另外，Hexo 拥有自己的 [ESLint配置](https://github.com/hexojs/eslint-config-hexo)，因此请确保您的贡献能够通过 ESLint。
```
Comment (SukkaW) [inline source/zh-cn/docs/contributing.md]:
```suggestion
另外，Hexo 拥有自己的 [ESLint 配置](https://github.com/hexojs/eslint-config-hexo)，因此请确保您的贡献能够通过 ESLint。
```

**Review by stevenjoezhang — APPROVED** [review]

**Pull request merged**